src/models/service/about.rs

```diff
@@ -9,7 +9,7 @@ use serde::Serialize;
 /// {
 ///   "transactionServiceBaseUri": "https://safe-transaction.staging.gnosisdev.com/api/v1",
```

Comment:
Maybe we should use the production URL for documentation?

Reply:
Same as above, environment shouldn't matter for the sake of documentation, but I would add the `/chains/4/` or something like that to reflect what could actually be returned.